hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/metrics/ClusterInfoCollector.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.internal.util.phonehome.metrics;
+
+import com.hazelcast.instance.impl.Node;
+import com.hazelcast.internal.cluster.impl.ClusterServiceImpl;
+import com.hazelcast.internal.util.phonehome.MetricsCollector;
+
+import java.lang.management.ManagementFactory;
+import java.lang.management.RuntimeMXBean;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ClusterInfoCollector implements MetricsCollector {
+
+    private static final int A_INTERVAL = 5;
+    private static final int B_INTERVAL = 10;
+    private static final int C_INTERVAL = 20;
+    private static final int D_INTERVAL = 40;
+    private static final int E_INTERVAL = 60;
+    private static final int F_INTERVAL = 100;
+    private static final int G_INTERVAL = 150;
+    private static final int H_INTERVAL = 300;
+    private static final int J_INTERVAL = 600;
+
+    public String convertToLetter(int size) {
```

Comment:
This is a duplicate of the same method in `BuildInfoCollector`. Please extract it into a common place. I suggest extracting it as a static method of the `MetricCollector` interface. Since it is used by the implementations of the interface, it is a good-enough approach.